data-transformers/transformers/fabanTransformer.py

```diff
@@ -0,0 +1,285 @@
+    
+    for path in minioPaths:
+        if "summary.xml" in path:     
+            fileName = path.split("/")[-1]
+            host = "aggregate"
+            if len(fileName.split(".")) > 2:
+                host = fileName.split(".")[2]
+            
+            data = getFromMinio(minioHost, minioPort, accessKey, secretKey, bucket, path)
+            #f = open(path, 'r')
+            data = f.read()
+            dataXML = xml.fromstring(data)
+            
+            def fA():
+                query = createRunInfoQuery(dataXML, trialID, experimentID, host)
+                query = sc.parallelize([query])
+                query.saveToCassandra(cassandraKeyspace, "faban_run_info", ttl=timedelta(hours=1))
+            
+            def fB():
+                query = createDriverSummaryQuery(dataXML, trialID, experimentID, host)
+                query = sc.parallelize(query)
+                query.saveToCassandra(cassandraKeyspace, "faban_driver_summary", ttl=timedelta(hours=1))
+                
+            def fC():
+                query = createDriverMixQuery(dataXML, trialID, experimentID, host)
+                query = sc.parallelize(query)
+                query.saveToCassandra(cassandraKeyspace, "faban_driver_mix", ttl=timedelta(hours=1))
+                
+            def fD():
+                query = createDriverResponseTimesQuery(dataXML, trialID, experimentID, host)
+                query = sc.parallelize(query)
+                query.saveToCassandra(cassandraKeyspace, "faban_driver_response_times", ttl=timedelta(hours=1))
+                
+            def fE():
+                query = createDriverDelayTimesQuery(dataXML, trialID, experimentID, host)
+                query = sc.parallelize(query)
+                query.saveToCassandra(cassandraKeyspace, "faban_driver_delay_times", ttl=timedelta(hours=1))
+                
+            def fF():
+                query = createDriverCustomStatsQuery(dataXML, trialID, experimentID, host)
+                query = sc.parallelize(query)
+                query.saveToCassandra(cassandraKeyspace, "faban_driver_custom_stats", ttl=timedelta(hours=1))
+                
+            tA = threading.Thread(target=fA)
+            tB = threading.Thread(target=fB)
+            tC = threading.Thread(target=fC)
+            tD = threading.Thread(target=fD)
+            tE = threading.Thread(target=fE)
+            tF = threading.Thread(target=fF)
+            tA.start()
+            tB.start()
+            tC.start()
+            tD.start()
+            tE.start()
+            tF.start()
+            tA.join()
+            tB.join()
+            tC.join()
+            tD.join()
+            tE.join()
+            tF.join()
+        
+        elif "detail.xan" in path:
+            fileName = path.split("/")[-1]
```

Comment:
This check is equal to the one at line https://github.com/benchflow/data-transformers/pull/68/files#diff-f378a6609614365d2cc18cb7277f5075R210. Place it in a method.